### PR TITLE
Vampire gui minor fix

### PR DESCRIPTION
**Pull request by Kaffe-work**
## About The Pull Request

Moves the GUI element for checking your blood level from a GUI button into a status bar tab 

## Why It's Good For The Game

Sanity check: is it reasonable to check a health/mana/resource element through a button? no. Moved it into a sidebar instead. Might also add a actual gui element, like xeno bar for measuring blood level in the future, some commented out code exists for this. 

**Also:** 
Changed blood loss to 1/3 of what it is now - playing vampire is nearly impossible unless you got monkey/genetics access. Given that it has neither antag status, large benefits, or access, i think this is not unreasonable. 

## Changelog
:cl:IradT
tweak: moved vampire blood counter into status bar
balance: vampire blood loss is now 1/3 of previous value
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species_types/vampire.dm [hunk 1]
```diff
@@ -139,3 +143,15 @@
...
+	var/obj/item/organ/heart/vampire/vessel = getorgan(/obj/item/organ/heart/vampire)
+	if(!vessel)
+		return 0
+	return blood_volume
+
+/mob/living/carbon/vampire/proc/updateBloodDisplay()
+	if(hud_used) //clientless aliens
+		 
+		hud_used.vampire_blood_display.maptext = "<div align='center' valign='middle' style='position:relative; top:0px; left:6px'><font color='red'>[round(getBlood())]</font></div>"
+*/
```
Comment (TheChosenEvilOne) [inline code/modules/mob/living/carbon/human/species_types/vampire.dm]: eww, commented out code.

Comment (Kaffe-work) [reply code/modules/mob/living/carbon/human/species_types/vampire.dm]: It's for the future, possible GUI changes

File: code/modules/mob/living/carbon/human/species_types/vampire.dm [hunk 2]
```diff
@@ -117,20 +120,20 @@
 
 #undef VAMP_DRAIN_AMOUNT
 
+
+/mob/living/carbon/Stat()
+	..()
+	if(statpanel("Status"))	
+		var/obj/item/organ/heart/vampire/darkheart = getorgan(/obj/item/organ/heart/vampire)
+		if(darkheart)
```
Comment (Jordie0608) [inline code/modules/mob/living/carbon/human/species_types/vampire.dm]: Check for vampire status with `isvampire()` instead of getting their heart type.

**Review by Jordie0608 — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/human/species_types/vampire.dm (hunk 2 above)
Comment (Kaffe-work) [reply code/modules/mob/living/carbon/human/species_types/vampire.dm]: having a vampire heart should allow you to see your blood level, no?

Comment (actioninja) [reply code/modules/mob/living/carbon/human/species_types/vampire.dm]: mechanic starts even on heart transplants not just with vampire species type.

Comment (Kaffe-work) [reply code/modules/mob/living/carbon/human/species_types/vampire.dm]: this is intended.

Comment (Jordie0608) [reply code/modules/mob/living/carbon/human/species_types/vampire.dm]: I wasn't aware, so it's fine.

**Pull request merged**